08_deploy_bmo.sh

```diff
@@ -6,6 +6,12 @@ source logging.sh
 source common.sh
 eval "$(go env)"
 
+# Set default value for provisioning interface
+PRO_IF=${PRO_IF:-ens3}
```

Comment:
This hijacks an existing variable from common.sh, and they're not the same thing, the existing one is the provisioning interface on the host running dev-scripts, but this one is the interface used for provisioning on the target RHCOS nodes.

Reply:
changed the name to PROV_IF to avoid overriding existing variables